cmd/repo_remove.go

```diff
@@ -25,8 +25,10 @@ func newRepoRemoveCmd(config *RootCommandConfig) *cobra.Command {
 	// initCmd represents the init command
 	var removeCmd = &cobra.Command{
 		Use:   "remove <name>",
-		Short: "Remove a configured Appsody repository",
-		Long:  ``,
+		Short: "Removes an Appsody repository.",
```

Comment:
Should be "Remove" rather than "Removes", to be consistent with "create" and other commands. Same for Long description.